pwndbg/commands/heap.py

```diff
@@ -1,46 +1,127 @@
-#!/usr/bin/env python
```

Comment:
Can you put the shebang and utf line back? It avoids Python2/3 issues in weird cases